cli/commands/hcl/format/cli.go

```diff
@@ -0,0 +1,92 @@
+package format
+
+import (
+	"github.com/gruntwork-io/terragrunt/cli/commands/common/runall"
+	"github.com/gruntwork-io/terragrunt/cli/flags"
+	"github.com/gruntwork-io/terragrunt/internal/cli"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+const (
+	CommandName      = "format"
+	CommandNameAlias = "fmt"
+
+	FileFlagName       = "file"
+	ExcludeDirFlagName = "exclude-dir"
+	CheckFlagName      = "check"
+	DiffFlagName       = "diff"
+	StdinFlagName      = "stdin"
+
+	DeprecatedHclfmtFileFlagName        = "hclfmt-file"
+	DeprecatedHclfmtcExcludeDirFlagName = "hclfmt-exclude-dir"
+	DeprecatedHclfmtStdinFlagName       = "hclfmt-stdin"
+	DeprecatedCheckFlagName             = "check"
```

Comment:
What's happening here? Why aren't we just using the `CheckFlagName` constant later?

Reply:
Good question. The reason why we declare multiple constants with the same value is because if we want to change the value of the current flag `CheckFlagName` to `new-check` and we use the same constant for the deprecated flag, then it will change as well so instead of the deprecated flag `terragrunt-cehck` it will be `terragrunt-new-check`. I thought there was no point in creating constants for legacy flags, instead specifying them as plain text where needed. for exmple, replace 

```go
flags.WithDeprecatedNames(terragruntPrefix.FlagNames(DeprecatedCheckFlagName), terragruntPrefixControl)),
```

with

```go
flags.WithDeprecatedNames(terragruntPrefix.FlagNames("check"), terragruntPrefixControl)),
``` 

and remove everywhere all `Deprecated*FlagName` constants. Are you agree?

Reply:
Personally, I would go with:

```go
flags.WithDeprecatedNames(terragruntPrefix.FlagNames(CheckFlagName), terragruntPrefixControl)),
```

The logic that you're doing is using a different prefix, so you can keep trying to leverage the standard constant for the name of the flag, but just use the different prefix in the arguments for `WithDeprecatedNames`.

I don't mind just hard-coding the legacy flag names, though. Do whatever makes the most sense to you.

Reply:
If you don't mind, I prefer to remove all `Deprecated*FlagName` constants. Since we don't use them for references from other parts of the code (logs, error messages, ...), and to be sure that changing the current flag name constant won't affect them, I hardcode the flag names. And with the reduction of lines in the file, it can also positively affect the readability of the code.